packages/markers/src/browser/problem/problem-decorator.ts

```diff
@@ -129,11 +129,19 @@ export class ProblemDecorator implements TreeDecorator {
         const icon = this.getOverlayIcon(marker);
         const color = this.getOverlayIconColor(marker);
         const priority = this.getPriority(marker);
+        //const isResource = Array.from(this.problemManager.getUris()).some(m => m === marker.uri);
+        //const problemStat = this.problemManager.getProblemStat();
         return {
             priority,
             fontData: {
                 color,
             },
+            tailDecorations: [
+                {
+                    data: 'allo',
+                    color
+                }
+            ],
             iconOverlay: {
                 position,
                 icon,
```

Comment:
We should revert the changes when upstreaming.
Alternatively, we can have a separate commit which adds this data for test purposes (we can drop the commit when it would be ready to merge).